lib/manager/nuget/extract.spec.ts

```diff
@@ -8,7 +9,7 @@ describe('lib/manager/nuget/extract', () => {
     let config: ExtractConfig;
     beforeEach(() => {
       config = {
-        localDir: upath.resolve('lib/manager/nuget/__fixtures__'),
+        localDir: path.resolve('lib/manager/nuget/__fixtures__'),
```

Comment:
@viceice weird.. this was the problem. Can you test on Windows that this returns different for `upath` 2.0.0 vs 2.0.1? I suspect its problem is it then causes problems with `find-up`

Reply:
🤔 tests pass on windows

Reply:
@viceice I mean, can you run some JS manually to verify that `upath.resolve('lib/manager/nuget/__fixtures__')` resolves differently in 2.0.0 vs 2.0.1 ?

Reply:
just checked on windows. resolved both times to same path